lib/src/data/repository/genre_repo.dart

```diff
@@ -1,18 +1,16 @@
 import 'dart:convert';
 import 'package:flutter/services.dart';
-import '../model/genre_data.dart';
-import '../utils/strings.dart';
-import './interface/genre_interface.dart';
+import '../../domain/entity/genre_data.dart';
+import '../../core/utils/strings.dart';
+import '../../domain/repository/interface/genre_interface.dart';
 
 class GenreRepository extends GenreInterface {
   List<Genre> parsedGenresList = [];
 
-  Future<List<Genre>> parseGenres() async {
+  static Future<List<Genre>> parseGenres() async {
```

Comment:
You don't need this method to be static. Also I'd mark this as private

Reply:
Done.